src/molecules/formfields/TextArea/text_area.module.scss

```diff
@@ -0,0 +1,25 @@
+@import '../shared/formfields.module.scss';
```

Comment:
I introduced this pattern with the FormField refactor. To move forward include this for now. 
I'll look into finding a permanent solution `@import formfield.module`